libs/dh/metering-point/feature-overview/src/utils/dh-format-number.ts

```diff
@@ -0,0 +1,25 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { formatNumber } from '@angular/common';
+
+import { WattSupportedLocales } from '@energinet-datahub/watt/date';
+
+export function dhFormatNumber(value: number | string, locale: WattSupportedLocales): string {
```

Comment:
Suggestion: So its not used for anything else, maybe we should call it formatMeasurementNumber

Reply:
Good point. I've pushed a new commit.